src/libaktualizr/package_manager/dockerappmanager.cc

```diff
@@ -38,6 +38,16 @@ struct DockerApp {
     return std::system(cmd.c_str()) == 0;
   }
 
+  void remove() {
+    auto bin = boost::filesystem::canonical(compose_bin).string();
+    std::string cmd("cd " + app_root.string() + " && " + bin + " down");
+    if (std::system(cmd.c_str()) == 0) {
+      boost::filesystem::remove_all(app_root);
+    } else {
+      LOG_ERROR << "Unable to shutdown DockerApp: " << app_root;
```

Comment:
Shutdown might be a bit confusing to the user, maybe just say that docker failed to bring the app down?

Reply:
ack